eng/pipelines/common/templates/wasm-library-aot-tests.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       shouldRunSmokeOnly: ${{ parameters.shouldRunSmokeOnly }}
       shouldContinueOnError: ${{ parameters.shouldContinueOnError }}
       scenarios:
-        - ${{ if (parameters.scenarios != '') }}:
+        - ${{ if eq(parameters.scenarios, '') }}:
```

Comment:
@ilonatommy should probably be `ne` ?

Reply:
I'm also not sure it works that way since we're expecting an array. I see other .yml define the parameter like this:
`scenarios: []`

Reply:
so I think in our case we want:

```yml
${{ if ne(parameters.scenarios[0], '') }}:
  scenarios: ${{ parameters.scenarios }}
${{ else }}:
  scenarios:
    - ${{ if eq(platform, 'browser_wasm') }}:
      - WasmTestOnV8
    - ${{ if eq(platform, 'browser_wasm_win') }}:
      - WasmTestOnChrome
    - ${{ if or(eq(platform, 'wasi_wasm_win'), eq(platform, 'wasi_wasm')) }}:
      - WasmTestOnWasmtime
```

Reply:
No all. Some templates have scenarios as string, e.g.
https://github.com/dotnet/runtime/blob/16a38c14d19fb9b4cf431d19ea682824e1e2f297/eng/pipelines/common/templates/runtimes/send-to-helix-step.yml#L16

but we can change it to array, I should not hurt.